vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt

```diff
@@ -285,6 +286,31 @@ class UiAllScreensSanityTest {
         pressBack()
     }
 
+    private fun navigateToRoomParameters() {
+        // Room addresses
+        clickListItem(R.id.roomSettingsRecyclerView, 4)
+        onView(isRoot()).perform(waitForView(withText(R.string.room_alias_published_alias_title)))
+        pressBack()
+
+        // Room permissions
+        clickListItem(R.id.roomSettingsRecyclerView, 6)
+        onView(isRoot()).perform(waitForView(withText(R.string.room_permissions_title)))
+        clickOn(R.string.room_permissions_change_room_avatar)
+        clickDialogNegativeButton()
+        // Toggle
+        clickOn(R.string.show_advanced)
+        clickOn(R.string.hide_advanced)
+        pressBack()
+
+        // Room history readability
+        clickListItem(R.id.roomSettingsRecyclerView, 8)
+        pressBack()
+
+        // Room access
+        clickListItem(R.id.roomSettingsRecyclerView, 10)
```

Comment:
Oh, I am surprised with 4, 6, 8, 10... Don't we have a better solution?

Reply:
Yes, that's really ugly... Using R.string.x to click on item based on the text does not work well, the list does not scroll automatically. And as we have separator items in the list, it increases 2 by 2...